lerna.json

```diff
@@ -2,7 +2,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.4.4",
+  "version": "1.4.5-alpha.0",
```

Comment:
I assume we'll be coming back here to update this version? Are we going to push this up, then ask MCC to pull this version down, test, then redploy? Then after verification, update this version? 

Asking because the changelog reflects a 1.5.0 release, which this PR is not.

Reply:
Yes, I am doing a pre-release "1.4.5-alpha.0", and thave MCC test the pre-release version, and once it's validate, it will update to v1.5.0 official release on this PR

Reply:
If this going to be 1.5.0 then would we want this to be `1.5.0-alpha.0`?